usecases/sharding/remote_index_incoming.go

```diff
@@ -92,8 +92,8 @@ func NewRemoteIndexIncoming(repo RemoteIncomingRepo) *RemoteIndexIncoming {
 func (rii *RemoteIndexIncoming) PutObject(ctx context.Context, indexName,
 	shardName string, obj *storobj.Object,
 ) error {
-	index := rii.repo.GetIndexForIncoming(schema.ClassName(indexName))
-	if index == nil {
+	index, err := rii.repo.GetIndexForIncoming(schema.ClassName(indexName))
+	if err != nil {
 		return errors.Errorf("local index %q not found", indexName)
```

Comment:
I agree I think having the error is a beneficial reminder to check for it

Reply:
I think returning an error for a non-existent index may not provide significant benefits, especially if we need to check for a non-nil pointer regardless. I am not opposed to this change but I would avoid it because it's not directly related to Raft, and altering it could lead to conflicts with the master branch